package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@modusoperandi/licit",
-  "version": "0.0.24",
+  "version": "0.0.25",
```

Comment:
**DO NOT** modify the version of package.json without making the exact same change in package-lock.json.  Better to leave unchanged until release is created.